app/repositories/payment-repository.js

```diff
@@ -16,7 +18,43 @@ export async function set (reference, data, frn) {
 export async function updatePaymentResponse (reference, status, paymentResponse) {
   const { models } = dataModels
   return models.payment.update(
-    { status, paymentResponse },
+    { status, paymentResponse, frn: paymentResponse.frn },
     { where: { applicationReference: reference } }
   )
 }
+
+export async function getPendingPayments () {
+  const { models } = dataModels
```

Comment:
Rather than having this in each function, could we destructure it once at the top of the file? And then each function can access models.